docs/conf.py

```diff
@@ -76,6 +80,20 @@
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = False
 
+# -- Extensions Managements -----------------------------------------------
+
+# sphinx.ext.extlinks: Markup to shorten external links
+#
+# Use it as :enos_src:`file/to/src.py`
+# See, http://www.sphinx-doc.org/en/stable/ext/extlinks.html
+extlinks = {
+  # Link to github sources of enos
+  'enos_src': ('https://github.com/BeyondTheClouds/enos/blob/' +
+               hash_version + '/%s', ''),
+  # Link to github raw sources of enos
+  'enos_cdn': ('https://raw.githubusercontent.com/BeyondTheClouds/enos/' +
```

Comment:
Can we use it for the `reservation.yaml.sample`

Reply:
By the way I just remember that one day I used this : 

![image](https://user-images.githubusercontent.com/2486813/29935705-089c5d10-8e80-11e7-9eb9-d2637d724d91.png)

The variable was defined in the `rst_epilog`
http://docutils.sourceforge.net/docs/ref/rst/directives.html#parsed-literal

Reply:
It works like a charm with `parsed-literal`. I updated the curl command in [installation.rst](https://github.com/BeyondTheClouds/enos/pull/191/files#diff-548eb9ce4cac8e13a1238ad703272931R50) with `:enos_cdn:`, so will never have to update links in the doc 👍.

Reply:
🎉   SO COOL !!